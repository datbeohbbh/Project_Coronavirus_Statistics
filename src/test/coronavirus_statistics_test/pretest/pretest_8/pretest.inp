5 20
? 5
? 5
q 1 3 40
? 4 1
? 6
q 3 3 8
q 1 2 31
q 2 2 9
? 4 1
? 4 2
q 3 2 3
? 4 2
? 4 1
q 3 3 24
q 1 4 41
? 7
? 4 2
? 4 2
? 4 3
? 4 2