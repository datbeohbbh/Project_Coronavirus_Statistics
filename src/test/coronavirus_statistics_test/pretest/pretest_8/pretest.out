Query ? 5 :
The number of safe cities is 5

Query ? 5 :
The number of safe cities is 5

Query ? 4 1 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 40
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 4 1 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 22
-> Has recovered : 9
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 32
-> Has recovered : 0
-> Was dead : 8

Query ? 4 2 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 32
-> Has recovered : 0
-> Was dead : 8

Query ? 4 1 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 19
-> Has recovered : 9
-> Was dead : 3

Query ? 7 :
The city with the highest death is city with the index 3
Quantity: 32 people

Query ? 4 2 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 19
-> Has recovered : 9
-> Was dead : 3

Query ? 4 2 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 19
-> Has recovered : 9
-> Was dead : 3

Query ? 4 3 :
The answer is the city with the index 4 
- Information of the city 4 -
-> Number of current patients : 41
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 19
-> Has recovered : 9
-> Was dead : 3

